client/src/components/Home/PartnersSection/PartnersSection.css

```diff
@@ -125,14 +125,14 @@ text-decoration: none;
       }
 }
 
-@media(min-width:1600px) {
+@media(min-width:1400px) {
       .container.partners-section {
             max-width: 85vw;
       }
 
       .partners-logo, .logo-name {
             width: 25vw;

```

Comment:
A bit of minor change and you are good to go, on hovering the sponsor's name is touching the edges which doesn't look good, so make the logo-name width:23vw or any optimum value which appears good to you.
(For reference hover over IEEE logo).